### ISSUE TITLE
Unclear wording in "Who do you think you are, War?"

### ISSUE THREAD

**Issue by nalzok** [NetHack/NetHack#958]
The following dialogue is ambiguous

https://github.com/NetHack/NetHack/blob/2511637169eaf54f51acdd95eb064beebcf29e74/src/sounds.c#L1088

However, it is clear that "War" is the player character

https://github.com/NetHack/NetHack/blob/2511637169eaf54f51acdd95eb064beebcf29e74/include/monsters.h#L2673

It probably makes more sense the reword the dialogue to "War, who do you think you are?" or maybe "War? What are you doing!"

**Comment by entrez** [comment]
I think the ambiguity is intentional.  At minimum it's common to interpret it as being intentional, and I think the mystique and ambiguity of how the hero's real identity is presented to the player is fun.

**Comment by copperwater** [comment]
I agree with entrez and don't think the messages should be changed, but it _would_ be funny to have someone say "War! What are you good for?"

**Comment by jerk-o** [comment]
Maybe the other three horsemen could say, "War! What are you good for?" as a "hint" that they recognize the player as the fourth horseman.
